Cdms.Backend.Data/Cdms.Backend.Data.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MongoDB.Driver" Version="3.0.0" />
```

Comment:
Are we sure this is a good idea on CDP?

Reply:
Version 3 introduces breaking changes, and JsonApi doesn't work with V3 at the moment :(